pkg/scheduler/schedule_one_test.go

```diff
@@ -3404,6 +3405,86 @@ func TestPreferNominatedNodeFilterCallCounts(t *testing.T) {
 	}
 }
 
+// Benchmark_findNodesThatFitPod is a performance benchmark for findNodesThatFitPod,
+// simulating a scenario with 5000 nodes and 5000 pods. at the same time, each pod corresponds to a node one by one
+// and the corresponding node is selected for each pod concurrently to test
+// the performance of findNodesThatFitPod under high concurrency
+func Benchmark_findNodesThatFitPod(b *testing.B) {
```

Comment:
can you change this to test at `scheduleOne` level instead?

Reply:
Also, maybe something like `Benchmark_scheduleOne_onePodPerNode`

Reply:
ok, I'll give it a try

Reply:
done